src/components/AssetSelection/components/AssetMenuButton.tsx

```diff
@@ -51,7 +51,7 @@ export const AssetMenuButton = ({
       isLoading={isLoading}
       {...buttonProps}
     >
-      <Flex alignItems='center' gap={2} width='100%' overflow='hidden'>
+      <Flex alignItems='center' gap={2} width='100%' overflow='visible' mx={1}>
```

Comment:
Mmmh, pending runtime test but It might break a few things

Reply:
Finally it looks fine, I think it's because back in the past we didn't have the overflow hidden on the span children